adal/src/main/java/com/microsoft/aad/adal/ADALError.java

```diff
@@ -314,7 +314,7 @@
      * Failed to use AndroidKeyStore.
      */
     ANDROIDKEYSTORE_FAILED("Failed to use AndroidKeyStore"),
-    
```

Comment:
This _is_ cleanup. There are four whitespace characters preceding the carriage return on each of these lines.

![screen shot 2017-01-11 at 10 08 26 am](https://cloud.githubusercontent.com/assets/990063/21860630/577d5cfa-d7e6-11e6-9aff-710c4afddf7d.png)